src/pc/1.14/ChunkColumn.js

```diff
@@ -197,6 +199,14 @@ module.exports = (Block, mcData) => {
       return this.sectionMask
     }
 
+    getMaskArray () {
+      return [this.getMask()]
+    }
+
+    countVerticalSections () {
```

Comment:
this should be added to api doc if it's useful

Reply:
Again I would much rather just remove these methods, but they were present in 1.16 so for API consistency they're present here

Reply:
same answer as other method, it was added in this pr